NebulaWorld/SimulatedWorld.cs

```diff
@@ -150,6 +150,25 @@ public static void UpdateRemotePlayerAnimation(PlayerAnimationUpdate packet)
             }
         }
 
+        public static void UpdateRemotePlayerWarpState(PlayerUseWarper packet)
+        {
+            using (GetRemotePlayersModels(out var remotePlayersModels))
+            {
+                if(packet.PlayerId == 0) packet.PlayerId = 1; // host sends himself as PlayerId 0 but clients see him as id 1
```

Comment:
We should fix this instead if its the case. We might have some unknown bugs related to this issue.